### PR TITLE
feat(test/misc): utils for verifying sequelize has no running/pending queries

### PR DESCRIPTION
**Pull request by mickhansen**
Should help in identifying leaks, lets see.

### CODE REVIEW

**Comment by mbroadst** [comment]
I feel like this thing is including some unrelated commits?

File: test/integration/model.test.js [hunk 1]
```diff
@@ -299,7 +299,7 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         bCol: { type: Sequelize.STRING, unique: 'a_and_b' }
       });
 
-      User.sync({ force: true }).on('sql', _.after(2, _.once(function(sql) {
+      return User.sync({ force: true, loging: _.after(2, _.once(function(sql) {
```
Comment (mbroadst) [inline test/integration/model.test.js]: ```logging``` not ```loging``` :smile:

**Comment by mickhansen** [comment]
@mbroadst it turned into a fix the god damn test suite PR

**Comment by mbroadst** [comment]
@mickhansen haha, souunnnndds fun!

**Comment by mickhansen** [comment]
@mbroadst not really but needs to be done ;p i just look forward to the day when everything that isn't written by me is cleaned up ;p

**Comment by mbroadst** [comment]
LGTM, someone hug @mickhansen

**Comment by mickhansen** [comment]
That took about 3 hours more than i expected.

**Pull request merged**